src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
+Animal.alive = [];
```

Comment:
It's better to define `alive` inside `Animal` class as static property. This would help you to be sure that `Animal.alive` is always initialized before it's used.